packages/expo-media-library/src/MediaLibrary.ts

```diff
@@ -474,16 +483,21 @@ export const usePermissions = createPermissionHook<
 
 // @needsAudit
 /**
- * __Available only on iOS >= 14.__ Allows the user to update the assets that your app has access to.
+ * __Available only on iOS >= 14 and Android >= 14.__ Allows the user to update the assets that your app has access to.
```

Comment:
```suggestion
 * Allows the user to update the assets that your app has access to.
```

We can use the platform tags to indicate the required platform versions:
```
@platform android 14+
@platform ios 14+
```


In long term, we would like to show all supported platforms for every API entry. The process will be automated, based on platforms set for the page, but in cases like that we would need to add those params to overwrite the default value for the package anyway.

A bit more information about params we use in source comments:
* https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#general-approach